lib/bolt/transport/api/connection.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# This is a copy of the orchestrator transport connection, but without the
+# 'start_plan' call in the init function which is handled by the orchestrator
+# client
+module Bolt
+  module Transport
+    class Api < Base
```

Comment:
This shouldn't work! Even if it does we need to extend the same base class in both declarations.

Reply:
What shouldn't work? I'm not sure what you mean by 'both declarations'? I was following the pattern in https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/orch/connection.rb#L5 - let me know if you want to bluejeans about it.

Reply:
Actually, separately, this relies on https://github.com/puppetlabs/bolt/blob/master/lib/bolt/transport/orch.rb#L60 to create new connections. I think it'd be more clear to explicitly create the connection (`Bolt::Transport::Orch::Connection.new`) and overload the method in API to create an API connection. But maybe it just adds unnecessary complexity and copy-pasting?